ddtrace/tracer.py

```diff
@@ -375,7 +375,7 @@ def configure(
         else:
             # No URL parts have updated and there's no previous writer to
             # get the URL from.
-            url = None  # type: ignore
```

Comment:
mypy fix, I can pull this out to another PR

Reply:
Fixed upstream in #3074